lab-cameron/routes/pokemon-routes.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const createError = require('http-errors');
+const pokemonCtrl = require('../controllers/pokemon-controller');
+
+module.exports = function(router) {
+  router.get('/api/pokemon/:id', (req, res) => {
+    if (!req.params.id) return res.status(400).send(createError(400, 'bad request'));
+    pokemonCtrl.fetchPokemon(req.params.id)
+    .then(pokemon => {
+      console.log(pokemon);
+      res.json(pokemon);
+    })
+    .catch(err => res.status(400).send(err.message));
+  });
+  router.get('/api/pokemon', (req, res) => {
+    pokemonCtrl.gottaFetchEmAll()
+    .then(pokemon => {
+      console.log(pokemon);
```

Comment:
**TODO FOR LOST POINTS**

Take out stray console.logs before submission, or manage them on another branch and submit a clean branch to me.